release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -124,6 +124,24 @@ For {product-title} {product-version}, you can quickly install a cluster on Amaz
 
 For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Intall a cluster quickly in AWS Local Zones].
 
+[id="ocp-4-14-install-update-cco-manual-enhancements"]
+==== Simplified installation and update experience for clusters with manually maintained cloud credentials
+
+This release includes changes that improve the experience of installing and updating clusters that use the Cloud Credential Operator (CCO) in manual mode for cloud provider authentication. The following parameters for the `oc adm release extract` command simplify the manual configuration of cloud credentials:
+
+`--included`:: Use this parameter to extract only the manifests that your specific cluster configuration needs.
++
+If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` custom resources for any disabled components before installing or updating a cluster.
```

Comment:
```suggestion
If you use cluster capabilities to disable one or more optional components, you are no longer required to delete the `CredentialsRequest` CRs for any disabled components before installing or updating a cluster.
```

Reply:
^Since we've already used CR for custom resource in an earlier section, should we use it here?

Reply:
oh nice yeah :+1: